### ISSUE TITLE
Searching isn't as easy as everyone thinks.  I know I'm not stupid.  Please help me!

### ISSUE THREAD

**Issue by CerebralFreeze** [keepassxreboot/keepassxc#4568]
## Expected Behavior
When I search for entries, a few results will pop up which is great.  If I click almost anywhere in a selected entry row, it will open the entry's detailed window.  There, I can edit and look at all the fields and other info.  When I press escape, the search results are still there.  And, when I keep KeePass open, the search results stay open instead of reverting back to selecting My Entire List.

## Current Behavior
1. After selecting an entry within the search results, it won't go into the Details window for the entry.  Since I didn't double-click the Title but rather any of the other columns, KeePass will just close or go to the URL.  I wish that if I click most of the row, it will go into the Details window.  For example, it would be nice to click the Username and it would go into the Details window and select the username so I could change it.  It would be a very smooth transition.
2. This is more annoying.  When I press Escape when I'm in an entry's Details window, it won't go back to my search results.  It'll revert to the original screen in which the main password database is selected.  It would be very nice if the search result is where I am returned to.  I frequently make a mistake and pick the incorrect entry.  If I press Escape, I'll have to do another search to bring up the same search results.  What's even more annoying is that sometimes, the search results are shown and KeePass will just automatically switch back to its original screen (selecting the main top database and showing other entries).  I don't know why it does this so secretively.  I don't know why she hates me.
3. How does KeePass select the entry to go into after I press enter, after entering the search terms?  When I press enter after putting in a search term like "email", it'll go into the Details window of a random entry.  It should just show the search results with maybe the first selected.  Pressing entering again will select the first entry and open its Details window.

## Possible Solution
Basically, kick the search function's butt to function normally.  Don't allow it to talk back.

## Context
What unique circumstances has this led to?  Well, this has led to rashes and hives, all over my body.  And, I can't be positive but I think it's caused me to contract the Coronavirus.  This is a very serious matter.  I'm not going to lay blame yet but I'm going to be watching very closely... 

## Debug Info
KeePassXC - Version 2.5.3
Revision: f8c962b

Qt 5.13.2
Debugging mode is disabled.

Operating system: Windows 10 (10.0)
CPU architecture: x86_64
Kernel: winnt 10.0.18362

Enabled extensions:
- Auto-Type
- Browser Integration
- SSH Agent
- KeeShare (signed and unsigned sharing)
- YubiKey

Cryptographic libraries:
 libgcrypt 1.8.5

**Comment by CerebralFreeze** [comment]
Thank you for the changes in title and label!  They are perfect!

**Comment by grravity** [comment]
Searching on Windows KPXC is more of a PITA than searching on Linux KPXC, I fail to understand why they are two different layouts?

**Comment by phoerious** [comment]
The layout on Windows and Linux are exactly the same.

**Comment by grravity** [comment]
@phoerious Weird, my Windows install does not have a search box.

**Comment by phoerious** [comment]
Make your window larger.

**Comment by droidmonkey** [comment]
Did you disable your toolbar?
